src/serac/numerics/equation_solver.cpp

```diff
@@ -868,6 +878,20 @@ std::pair<std::unique_ptr<mfem::Solver>, std::unique_ptr<mfem::Solver>> buildLin
     case LinearSolver::GMRES:
       iter_lin_solver = std::make_unique<mfem::GMRESSolver>(comm);
       break;
+#ifdef SERAC_USE_PETSC
+    case LinearSolver::PetscCG:
```

Comment:
Should this be `LinearSolver::PETScCG`?

Reply:
I vote no, since that would be different than the MFEM/PETSc style.